client/src/document/index.scss

```diff
@@ -328,7 +328,9 @@ iframe {
     background: #fff;
     border: 1px solid var(--border-primary);
     border-radius: var(--elem-radius);
-    width: 100%;
+    box-sizing: content-box;
+    padding: 1rem;
+    width: calc(100% - 2rem);
```

Comment:
```suggestion
    width: calc(100% - 2rem - 2px);
```
To have accurate alignment we need to subtract `-2px` (for the borders) as well:

![2pxExtra](https://user-images.githubusercontent.com/87750369/188878138-f0d53f1e-2f4a-4311-9853-594529f04910.png)

Reply:
Good catch, thank you!